citus.spec

```diff
@@ -43,6 +43,7 @@ make %{?_smp_mflags}
 # Install documentation with a better name:
 %{__mkdir} -p %{buildroot}%{pginstdir}/doc/extension
 %{__cp} README.md %{buildroot}%{pginstdir}/doc/extension/README-%{sname}.md
+%{__cp} NOTICE %{buildroot}/usr/NOTICE
```

Comment:
Several issues with this:

- the folder `%{buildroot}/usr` may not exist
- we would not want to place a file with a generic name in a directory that is not owned by citus

```suggestion
%{__cp} NOTICE %{buildroot}%{pginstdir}/doc/extension/NOTICE-%{sname}
```

Reply:
Fixed
![image](https://user-images.githubusercontent.com/1596331/142406829-7fca78a9-a131-41ff-9f9a-b3fc6bbabb54.png)